src/helpers/getByAPI.js

```diff
@@ -107,11 +107,10 @@ const getTextInputNodeByPlaceholderText = (node, placeholder) => {
 const getTextInputNodeByDisplayValue = (node, value) => {
   try {
     const { TextInput } = require('react-native');
+    const nodeValue = node.props.value || node.props.defaultValue;
```

Comment:
I think we want to be explicit here and check if `node.props.value === undefined`

Reply:
indeed that would be better, which value do we use if value is empty string?

Reply:
I _think_ it's always gonna be a string for Text, but the only way to be sure would be to test it. 

Nevertheless, if the `value` is `""` the defaultValue is no longer used as the `value` is set to not-undefined

Reply:
Oh alright I did not know that, I'll make the changes

Reply:
Flow does not let me put 0 in the value prop of the TextInput, it wants a string. Should I ignore it or do we consider it's fine? (especially since we'll now be checking for undefined and not false values)

Reply:
No need to test the number then :)